src/main/resources/application-dev.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# The "-dev" profile is for running the API service locally against pre-existing
+# docker containers that are listening on known port numbers for hmpps-auth (8090) and
+# a postgres database (5432).
+#
+
+# Most values are inherited from - application.yml - except the following:
+
+spring:
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          public-key-location: classpath:local-public-key.pub
+
```

Comment:
Probably need a copy of this file `local-public-key.pub` from CVL? It goes into `src/main/resources`

Reply:
I've addressed that one, also fixed an indenting issue in the application.yml